scripts/app/convenient-effects-controller.js

```diff
@@ -337,12 +337,31 @@ export default class ConvenientEffectsController {
 
   /**
    * Handles starting the drag for effect items
+   * For non-nested effects, populates the dataTransfer with Foundry's expected
+   * ActiveEffect type and data to make non-nested effects behave as core does
    *
    * @param {DragEvent} event - event that corresponds to the drag start
    */
   onEffectDragStart(event) {
     const effectName = event.target.dataset.effectName;
-    event.dataTransfer.setData('text/plain', JSON.stringify({ effectName }));
+
+    const effect = game.dfreds.effectInterface.findEffectByName(effectName);
+
+    // special handling for nested effects
+    if (effect.nestedEffects.length) {
+      event.dataTransfer.setData('text/plain', JSON.stringify({
+        effectName,
+      }));
+      return;
+    }
+
+    // otherwise use core default format
+    const effectData = effect.convertToActiveEffectData();
+
+    event.dataTransfer.setData('text/plain', JSON.stringify({
```

Comment:
Changing the drop data also affects the `onDropOntoFolder` function, which is used when dragging and dropping an effect onto the Favorites folder. So that function would likely need updated as well.

Similarly, this also broke the dragging and dropping onto a hotbar slot (in main.js):

```js
Hooks.on('hotbarDrop', (bar, data, slot) => {
  const macroHandler = new MacroHandler();
  macroHandler.createMacro(data, slot);
});
```

So that would also need updated. They could probably both still work in such a way that the effectName is still available for these situations, or if you update the parsing of the event data so the actual `Effect` can be found still.

Reply:
Just to further clarify, it might be enough to rely on the label of the active effect data rather than explicitly expect an `effectName` in that data.

Reply:
Re-adding `effectName` and handling nested effects specially seems like the simplest play here IMO.

I tested:
- Favorites Folder drop ✅ 
- Macrobar drop ✅ 
- Actor Sheet drop ✅ (both normal and nested effects)
- Item Sheet drop (via Drop effects on items module)